src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp

```diff
@@ -1328,9 +1224,37 @@ void jit_power_static_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs,
 
         const std::unordered_set<size_t> exclude = {src().getIdx(), dst.getIdx()};
         store_context(exclude);
-        for (auto i = 0; i < 4; i++) {
-            h->mov(s0, src().s[i]);
-            h->ldr(s1, table_val("power"));
+        const auto length = exec_prc_ == ov::element::f32 ? 4 : 8;
+        for (auto i = 0; i < length; i++) {
+            if (exec_prc_ == ov::element::f32) {
+                Xbyak_aarch64::VReg4S src2(get_from_dst ? out_vec_idxs[0] : in_vec_idxs[0]);
+                h->mov(s0, src2[i]);
+                h->ldr(s1, table_val2("power"));
+            } else if (exec_prc_ == ov::element::f16) {
+                Xbyak_aarch64::VReg8H src2(get_from_dst ? out_vec_idxs[0] : in_vec_idxs[0]);
+                Xbyak_aarch64::HReg h0(0);
+                h->mov(h0, src2[i]);
+                h->fcvt(s0, h0);
+
+                Xbyak_aarch64::HReg h1(1);
+                h->ldr(h1, table_val2("power"));
+                h->fcvt(s1, h1);
+            }
+
+            if (exec_prc_ == ov::element::f32) {
```

Comment:
Could you merge this conditional block with block on line #1229

Reply:
removed, thanks